filebeat/harvester/log.go

```diff
@@ -265,6 +267,6 @@ func (h *Harvester) close() {
 		h.file.Close()
 		logp.Debug("harvester", "Stopping harvester, closing file: %s", h.Path)
 	} else {
-		logp.Debug("harvester", "Stopping harvester, NOT closing file as file info not available: %s", h.Path)
+		logp.Warn("harvester", "Stopping harvester, NOT closing file as file info not available: %s", h.Path)
```

Comment:
we can try by not giving permissions for some file and do lsof. But prospector should have tripped before when getting stats.